### ISSUE TITLE
[Improvement] Tuya TRV06

### ISSUE THREAD

**Issue by umbertix** [jacekk015/zha_quirks#30]
Hi, first and foremost thx for this amazing work.

I recently bought a Tuya TRV06, the internal id that I pulled out is: `_TZE200_p3dbf6qs`, and I managed to get it somewhat working with the `ts0601_trv_me167`. This made the device from unknown to an actual TRV. I get the temperature reading correctly, both the "current" and the "desired" temperatures and the child lock.

This is already great but I know that the device has some extra functionalities that would like exposed if possible, like:

- Antifreeze
- Schedules
- Window opened

Unfortunetly I do not own the Tuya bridge to go round this, but I would happily provide logs or extra data points if are necessary.

Thx again

**Comment by jacekk015** [comment]
Antifreeze
Frost protection switch has been added.

Window opened
There's no such function on this TRV - according to Z2M implementation. Check the manual.
Even if it was there - that never works, unless you put a TRV in the freezer.

Schedules
Will not do this. Since every TRV has it's own approach to this work-hours needed are tremendous.
Tests and etc... without having that specific TRV in hands - remote DEBUG is not an easy option.
Secondly: ZHA/Home Assistant doesn't have any GUI control to put this there.
I did Schedule only for Maxsmart because I own few of them. Even that, since there's no GUI, it can only be managed by Cluster Attributes
![image](https://github.com/jacekk015/zha_quirks/assets/14165726/16d318f6-19f3-4ec3-a188-c0a2e94ad63e)
Even made - I'm not using it. It takes few seconds to go and reprogram it on the TRV device.
Mostly it's done 1 or 2 times in the TRV lifetime.
